providers/ses/src/lib/ses.provider.ts

```diff
@@ -85,6 +87,62 @@ export class SESEmailProvider implements IEmailProvider {
     };
   }
 
+  getMessageId(body: any | any[]): string[] {
+    if (Array.isArray(body)) {
+      return body.map((item) => item.mail.messageId);
+    }
+
+    return [body.mail.messageId];
+  }
+
+  parseEventBody(
+    body: any | any[],
+    identifier: string
+  ): IEmailEventBody | undefined {
+    if (Array.isArray(body)) {
+      body = body.find((item) => item.mail.messageId === identifier);
+    }
+
+    if (!body) {
+      return undefined;
+    }
+
+    const status = this.getStatus(body.eventType);
+
+    if (status === undefined) {
+      return undefined;
+    }
+
+    return {
+      status: status,
+      date: new Date(body.mail.timestamp).toISOString(),
+      externalId: body.mail.messageId,
+      row: body,
+    };
+  }
+
+  private getStatus(event: string): EmailEventStatusEnum | undefined {
+    switch (event) {
+      case 'Bounce':
+        return EmailEventStatusEnum.BOUNCED;
+      case 'Complaint':
+        return EmailEventStatusEnum.COMPLAINT;
+      case 'Delivery':
+        return EmailEventStatusEnum.DELIVERED;
+      case 'Send':
+        return EmailEventStatusEnum.SENT;
+      case 'Reject':
+        return EmailEventStatusEnum.REJECTED;
+      case 'Open':
+        return EmailEventStatusEnum.OPENED;
+      case 'Click':
+        return EmailEventStatusEnum.CLICKED;
+      case 'DeliveryDelay':
+        return EmailEventStatusEnum.DELAYED;
+      // case 'Subscription':
```

Comment:
Just remove it. If you want to leave a function comment before line 124 mentioning that `Subscription` event status is not considered for the reasons explained would be a bit more meaningful.